xbatcher/accessors.py

```diff
@@ -1,8 +1,23 @@
+from typing import Union
+
 import xarray as xr
 
 from .generators import BatchGenerator
 
 
+def _as_xarray_dataarray(
+    xr_obj: Union[xr.Dataset, xr.DataArray]
+) -> Union[xr.Dataset, xr.DataArray]:
```

Comment:
```suggestion
) -> xr.DataArray:
```
output should be a DataArray